src/WebJobs.Script/Workers/ProcessManagement/DefaultWorkerProcessFactory.cs

```diff
@@ -89,5 +94,21 @@ internal string SanitizeExpandedArgument(string envExpandedString)
             }
             return envExpandedString;
         }
+
+        internal void ApplyWorkerConcurrencyLimits(ProcessStartInfo startInfo)
+        {
+            // Set higher concurrency limits for Python and Powershell language workers for V4
+            string functionWorkerRuntime = startInfo.EnvironmentVariables.GetValue(RpcWorkerConstants.FunctionWorkerRuntimeSettingName);
+            if (string.IsNullOrEmpty(startInfo.EnvironmentVariables.GetValue(RpcWorkerConstants.PythonThreadpoolThreadCount)) &&
+                string.Equals(functionWorkerRuntime, RpcWorkerConstants.PythonLanguageWorkerName, StringComparison.OrdinalIgnoreCase))
+            {
+                startInfo.EnvironmentVariables[RpcWorkerConstants.PythonThreadpoolThreadCount] = RpcWorkerConstants.DefaultConcurrencyLimit;
```

Comment:
can these be exposed in worker.config.json instead of AppSettings?

Reply:
No, these settings are user-configurable per app. They are not new, they already exist and the language workers expect them.

Reply:
The DefaultConcurrencyLimit could be a part of worker.config, but adding them doesn't add any value as they can be overwritten by the Cx.